app/assets/stylesheets/marketing/modules/_variables.scss

```diff
@@ -15,7 +15,7 @@ $max-width: 				1001px;
  * Breakpoints
  */
 $short: 					new-breakpoint(max-width 22.5em 4);     // 360px
-$tall: 						new-breakpoint(max-width 47.8em 12);      // 768px
+$tall: 						new-breakpoint(max-width 52.94em 12);      // 847px
```

Comment:
I'm concerned that this will affect more than just the header wrapping. Maybe better to adjust nav one-off?